docs/USAGE.md

```diff
@@ -0,0 +1,137 @@
+# Usage
```

Comment:
It's a bit long and has a lot of nuanced details re. containerd and HostProcess containers. I think it might be easier to keep this in a separate doc and point to it. I like to keep READMEs a hub of pointers to relevant information.